data-processing/entities/definitions/commands/detect_definitions.py

```diff
@@ -453,26 +824,40 @@ def process(
         definition_texs: Dict[TermName, List[str]] = {}
         definition_texts: Dict[TermName, List[str]] = {}
         sources: Dict[TermName, List[str]] = {}
+        position_ratios: Dict[TermName, List[float]] = {}
+        section_names: Dict[TermName, List[str]] = {}
 
         # Associate terms with all definitions that apply to them.
         for term, definiendum_list in definiendums.items():
-            definition_ids[term] = [d.definition_id for d in definiendum_list]
+            definition_ids[term] = [
+                definiendum.definition_id for definiendum in definiendum_list
```

Comment:
hahaha okay I think for iterators this small (e.g., inline), single letter variable names are ok to use. There's no need to change theses though this may be useful to know for the future.